operator/hack/kind-up.sh

```diff
@@ -135,7 +141,16 @@ function kind::create_cluster() {
   mkdir -p "${KIND_CONFIG_DIR}"
   echo "Creating kind cluster ${CLUSTER_NAME}..."
   kind::generate_config
+  
+  # If KUBECONFIG is not already set (e.g., by the Makefile), set it to our default location
```

Comment:
Just fyi, ~/.kube/config is the defacto default without KUBECONFIG. New clusters get added there which can be good or bad. But you don't absolutely need to have KUBECONFIG set all the time.